Library/PhantomLib.Droid/PhantomLib.Droid.csproj

```diff
@@ -171,6 +171,12 @@
     <Reference Include="Xamarin.Forms.Xaml">
       <HintPath>..\..\packages\Xamarin.Forms.4.2.0.709249\lib\MonoAndroid90\Xamarin.Forms.Xaml.dll</HintPath>
     </Reference>
+    <Reference Include="Prism">
```

Comment:
I'd like to not have a dependency on Prism for this library. Prism is a bit heavy and there are some people that will not want to use it.